agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -166,6 +167,7 @@ public void map(SpanData span, Consumer<TelemetryItem> consumer) {
     consumer.accept(telemetryItem);
     exportEvents(
         span,
+        telemetryItem.getData().getBaseData() instanceof RequestData,
```

Comment:
I think it's the same, since we map spans to either requests or dependencies, I think I like instance of RequestData because this is how I am thinking about it after the discussion with Osvaldo:

> Application Insights "exceptions" are meant for top-level exceptions (exceptions on request telemetry or logged exceptions)

Reply:
does it imply that it can be MessageData too? 

// (exceptions on request telemetry or **logged exceptions**)

Reply:
that is in LogDataMapper:

https://github.com/microsoft/ApplicationInsights-Java/blob/e1b89d42a7cbb995748205c4ea3e96d57319e9e2/agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java#L62-L67

Reply:
(where we create either message or exception record, but not both)